zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DeletedDiskChecker.scala

```diff
@@ -36,12 +36,14 @@ object DeletedDiskChecker {
             case Left(e) =>
               val result = e match {
                 case ee: com.google.api.gax.rpc.PermissionDeniedException =>
-                  if (
-                    ee.getCause.getMessage.contains("Compute Engine API has not been used") || ee.getCause.getMessage
-                      .contains("This API method requires billing to be enabled")
-                  )
+                  if (ee.getCause.getMessage.contains("Compute Engine API has not been used")) {
                     if (isDryRun) F.unit else dbReader.updateDiskStatus(disk.id)
-                  else logger.error(e)(s"fail to get ${disk.diskName}")
+                  } else if (
+                    ee.getCause.getMessage
+                      .contains("This API method requires billing to be enabled")
+                  ) {
+                    logger.info(s"billing is disabled for ${disk.diskName}")
```

Comment:
If I remember correctly, when we (AoU and IA) met to talk about this change, I believe that we mentioned logging when billing is disabled might be useful instead of just removing the condition from the if statement. If anyone has different thoughts, please let me know.